src/Microsoft.Health.Dicom.Api/Controllers/ExtendedQueryTagController.cs

```diff
@@ -65,10 +66,24 @@ public async Task<IActionResult> PostAsync([Required][FromBody] IReadOnlyCollect
         _logger.LogInformation("DICOM Web Add Extended Query Tag request received, with extendedQueryTags {ExtendedQueryTags}.", extendedQueryTags);
 
         EnsureFeatureIsEnabled();
```

Comment:
just checking, do we still need this flag. This is On everywhere right?

Reply:
Yeah, it is enable everywhere today. I'll create a separate PR to remove this flag